src/runtime_src/xrt/device/halops2.cpp

```diff
@@ -25,121 +25,18 @@ operations(const std::string &fileName, void *fileHandle, unsigned int count)
   : mFileName(fileName)
   ,mDriverHandle(fileHandle)
   ,mDeviceCount(count)
-  ,mProbe(0)
```

Comment:
Can we get rid of halop2.{h,cpp} and halops2_static.cpp?

Reply:
Yes Soren, I will completely remove hal ops in my next PR.
For some 5 functions there is no 1-1 mapping functions which I can call as an alternate so did it this way.